org/cloudfoundry.yml

```
@@ -5787,6 +5788,7 @@ orgs:
         - joergdw
         - KevinJCross
         - silvestre
+        - marcinkubica
         members:
```

Unfortunately you do not yet meet the requirements for a maintainer as defined in https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#requirements-2 - Please add yourself to the `members` section

```suggestion
        members:
        - marcinkubica
```